models/repo.go

```diff
@@ -1599,9 +1610,10 @@ func SearchRepositoryByName(opts *SearchRepoOptions) (repos []*Repository, _ int
 	if err != nil {
 		return nil, 0, fmt.Errorf("Count: %v", err)
 	}
+	log.Info("Count %d", count)
```

Comment:
will remove sorry about that. I was using the logger before I figured out how to get delve to work with vscode and forgot about that line.